bugbot/crash/analyzer.py

```diff
@@ -717,9 +718,13 @@ def handler(search_resp: dict, data: list):
                         # Potential bad hardware crash, skip it.
                         continue
 
-                # TODO: Add a filter using the `possible_bit_flips_max_confidence`
-                # field to exclude bad hardware crashes. The filed is not available yet.
-                # See: https://bugzilla.mozilla.org/show_bug.cgi?id=1816669#c3
+                bit_flips_percentage = (
+                    facets["cardinality_possible_bit_flips_max_confidence"]["value"]
+                    / crash["count"]
+                )
+                if bit_flips_percentage >= 0.2:
```

Comment:
If 1 crash is by 100% a bit flip and 4 are not, the signature would get skipped.

Might this need future work to prevent the suppression of crash reporting if a signature with much bit flip crashes sees an increase in valid crashes from a regression?

Reply:
The current value (i.e. 20%) is suggested by @gabrielesvelto. We will monitor the crashes that we miss filing due to this threshold and adjust when necessary.